admission-webhook/deploy/deploy-gmsa-webhook.sh

```diff
@@ -104,7 +104,11 @@ main() {
     CREATE_SIGNED_CERT_SCRIPT=$(ensure_helper_file_present 'create-signed-cert.sh')
 
     if [ ! -x "$(command -v envsubst)" ]; then
-        fatal_error 'envsubst not found'
+        if hash apt-get 2>/dev/null; then
```

Comment:
Still not a huge fan of this. Running envsusbt in a container would be preferable. See https://github.com/kubernetes-sigs/windows-gmsa/pull/21#discussion_r340214748.

Reply:
Running envsubst from a docker container would increase the complexity quite a bit, since you would have to write all env vars to a file and also create temporary folders to mount into the container, but if you still prefer that over the implemented solution please let me know and I'll change it up.

Reply:
I don't think there's that much additional complexity involved? In particular, there shouldn't be any need to write env vars to a file?

Reply:
If I don't write them to a file, I would need to pass them individually to the docker run command through -e args, as well as writing them twice (once in the call to envsubst and once in the call to docker), unless we want to eliminate the call to envsubst and always use a docker container for running envsubst.

Reply:
Not necessarily:
```
maggy:tmp wk$ cat target.yml 
foo: ${FOO}
bar: ${BAR}
maggy:tmp wk$ CMD='envsubst < target.yml'
maggy:tmp wk$ env - PATH="$PATH" FOO='foo value' BAR='bar value' bash -c "$CMD"
foo: foo value
bar: bar value
maggy:tmp wk$ CMD='docker run --rm -v $(pwd):/tmp --env-file <(env) supinf/envsubst /tmp/target.yml'
maggy:tmp wk$ env - PATH="$PATH" FOO='foo value' BAR='bar value' bash -c "$CMD"
foo: foo value
bar: bar value
```